static/app/views/replays/detail/network/index.tsx

```diff
@@ -87,6 +104,22 @@ function NetworkList({replayRecord, networkSpans}: Props) {
     [handleMouseEnter, handleMouseLeave]
   );
 
+  useEffect(() => {
+    // Observe the network table for width changes
+    const observer = new ResizeObserver(() => {
+      // Recompute the column widths
+      multiGridRef.current?.recomputeGridSize({columnIndex: 1});
```

Comment:
We need to recompute the grid size of the `path` column if the container width has changed, so it can fill the remaining space